src/instructlab/data/generate.py

```diff
@@ -125,6 +125,15 @@
     "--model-family",
     help="Force model family to use when picking a generation template",
 )
+@click.option(
+    "--pipeline",
+    type=click.STRING,
+    default="simple",
```

Comment:
should this be a click.choice with the only option as simple? otherwise we will need to change flag type in the future.

Reply:
oh good call. i didn't realize that was an option, thanks

Reply:
updated to use click.choice, thanks for the suggestions